app/_src/kubernetes-ingress-controller/guides/migrating-from-ingress-to-gateway.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Migrating from Ingress to Gateway
+type: how-to
+purpose: |
+    How to migrate from Ingress API to Gateway API
+---
+
+## Prerequisites
+
+This guide will use git and [go][go] to download and execute the [`ingress2gateway`][ingress2gateway]
+tool.
+
+```bash
+git clone https://github.com/kong/ingress2gateway
+cd ingress2gateway
+git checkout preview
+make build
+export PATH=$PATH:$(pwd)
+```
+
+[go]: https://go.dev/dl/
+[ingress2gateway]: https://github.com/kubernetes-sigs/ingress2gateway
+
+## Convert all your yaml files
+
+In order to migrate your resources from ingress to gateway, you need all your ingress-based
+`yaml` manifests. Starting from them you can migrate to the new API by creating copies
+that replace the ingress resources with Gateway API ones.
+Now, use the `ingress2gateway` tool to create new manifests containing the gateway
+API configurations:
+
+1. Export your source and destination paths:
+
+    ```bash
+    export SOURCE_DIR=<your_source_directory>
+    export DEST_DIR=<your_destination_directory>
```

Comment:
```suggestion
    SOURCE_DIR=<your_source_directory>
    DEST_DIR=<your_destination_directory>
```
Since you're executing the subsequent commands in the same shell you don't need to `export`, but exporting doesn't harm, so for your own judge

Reply:
Today I learned! 🎓